docs/_static/spec/rasa.yml

```diff
@@ -920,6 +971,26 @@ components:
           example: 0.987232
       required: ["name"]
 
+    IntentInjectionRequest:
+      type: object
+      properties:
+        name:
+          description: >-
+            Name of the intent to be executed.
```

Comment:
Would change to just `description: Name of the intent to be executed.` (and the two other descriptions as well).